realm/realm-annotations-processor/src/main/java/io/realm/processor/Backlink.kt

```diff
@@ -83,6 +83,11 @@ class Backlink(clazz: ClassMetaData, private val backlinkField: VariableElement)
      */
     val sourceField: String? = backlinkField.getAnnotation(LinkingObjects::class.java)?.value
 
+    /**
+     * {@code true} if the parent link should be modeled as a single link instead of as a RealmResults.
+     */
+    val exposeAsRealmResults: Boolean = Utils.isRealmResults(backlinkField)
+
     val targetFieldType: String
         get() = backlinkField.asType().toString()
```

Comment:
`backlinkField` is also a val from the constructor.

Reply:
I think creating a backing property with `get` is a superfluous here. The value won't change given that `backlingField` is a `val`, so there is no need to compute `backlinkField.asType().toString()` every time we access `targetFieldType`. It could be done without the `get()`.

Reply:
Good point, might assign the value directly 👍